Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "uuid"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
We don't touch version number until preparing a release

Reply:
Done